co2eq/purchase/index.js

```diff
@@ -116,8 +117,16 @@ function correctWithParticipants(footprint, participants) {
   return footprint / (participants || 1);
 }
 
+export function convertTo2011Euro(amount, currency) {
+  const exchangeRate2011 = exchangeRates2011.rates[currency.toUpperCase()];
+  if (!exchangeRate2011 && exchangeRate2011 !== 0) {
```

Comment:
!= null is a better way to do this as it will check for both null and undefined. Or did I miss something?

Reply:
well we want to throw the error only if it's undefined or null not the other way around?

Reply:
ah yes 🤦 - then this is what I suggest:
```suggestion
  if (exchangeRate2011 == null) {
```
this will be true when `exchangeRate2011` is `null` or `undefined`